spello/symspell/symspell.py

```diff
@@ -213,7 +250,7 @@ def get_suggestions(self, string, silent=False):
             queue = queue[1:]
 
             # early exit
-            if ((self.verbose < 2) and (len(suggest_dict) > 0) and
+            if ((self.config.symspell_verbosity < 2) and (len(suggest_dict) > 0) and
```

Comment:
Should we leave in `verbose` as a parameter, in case older version is still being used / convenience if calling class in shell?

Maybe `self.verbose = self.config.symspell_verbosity or verbose` in init?